docs/tutorials/examples.md

```diff
@@ -100,22 +100,22 @@ Quantum circuit to prepare the BCS ground states for superconductors/superfluids
 
 ## Advanced Tutorials
 
-*    [Variational Algorithm](https://cirq.readthedocs.io/en/latest/tutorials/variational_algorithm.html)
+*    [Variational Algorithm](./variational_algorithm)
```

Comment:
Do we need an extension for this (ipynb)?  Same with below.  Your comment suggests we should add it.

Reply:
Yes. You can test this in the [github preview](https://github.com/quantumlib/Cirq/blob/e59015ee42620ab9221a9b2dda9191d05e015d7d/docs/tutorials/examples.md). This link doesn't work in GitHub unless you add `variational_algorithm.ipynb`
It will work on the site since we strip extensions anyway, but, as a general style, using the extension means the link works everywhere.

Reply:
Okay, thanks for the input - I agree, I'll change this back to .ipynb.
However, this will need a bit more work due to a bug in recommonmark - I'm running into https://github.com/readthedocs/recommonmark/issues/130. 

The current version of this page https://cirq.readthedocs.io/en/latest/docs/tutorials/examples.html is broken both on RTD and in devsite. I'm having promising results with [myst](https://myst-parser.readthedocs.io/en/latest/index.html)...